src/DeviceInterface/InterfacePrototype.js

```diff
@@ -379,18 +379,20 @@ class InterfacePrototype {
          * This is desktop only: was  it a private address? if so, save it with
          * the trigger 'emailProtection' so that native sides can use it
          */
-        if (isPrivateAddress && 'emailAddress' in data && data.emailAddress) {
+        if (isPrivateAddress) {
```

Comment:
The diff is messy unless you hide whitespace changes, but I'm just splitting this conditional so that `refreshAlias` is always called regardless of the need to store the data.